packages/codebytes/src/codeByteEditor.tsx

```diff
@@ -12,38 +12,28 @@ import { LanguageSelection } from './languageSelection';
 import { trackUserImpression } from './libs/eventTracking';
 import { CodeByteEditorProps } from './types';
 
-const editorStates = states({
-  isIFrame: { height: '100vh' },
-});
-
 const editorBaseStyles = system.css({
   border: 1,
   borderColor: 'gray-900',
   display: 'flex',
   flexDirection: 'column',
-  height: '25rem',
-  width: '43rem',
   overflow: 'hidden',
 });
-export interface EditorStyleProps
-  extends StyleProps<typeof editorBaseStyles>,
-    StyleProps<typeof editorStates> {}
 
-const EditorContainer = styled(Background)<EditorStyleProps>(
-  editorBaseStyles,
-  editorStates
+const EditorContainer = styled(Background)<StyleProps<typeof editorBaseStyles>>(
+  editorBaseStyles
 );
 
 export const CodeByteEditor: React.FC<CodeByteEditorProps> = ({
   text: initialText,
   language: initialLanguage,
   hideCopyButton = false,
-  isIFrame = false,
   snippetsBaseUrl,
   onEdit,
   onLanguageChange,
   onCopy,
   trackingData,
+  ...rest
```

Comment:
what if instead of rest, we pass in `style`? do we need anything else besides styles?

Reply:
copying over from slack... the rest syntax gives us the styled system from gamut for Background

Reply:
rest cant be anything because of typescript 
it can just be anything on CodeByteEditorProps (which will now extend BackgroundProps)

Reply:
That works, this is probably more extensible in the end, if we want to pass in something other than styles in the future (`ref`, styled components, etc)